tests/factory/test_factory_unused_options.py

```diff
@@ -0,0 +1,111 @@
+import copy
+import os
+
+import mozfile
+import pytest
+
+from mozdownload.utils import urljoin
+from mozdownload import FactoryScraper
+
+
+@pytest.mark.parametrize('scraper_type,builds,fname,arg', [
+    # ReleaseScraper
+    ('release', None, 'firefox-23.0.1.en-US.win32.exe', {
+        'version': '23.0.1',
+        'platform': 'win32',
+
```

Comment:
Maybe you can add a comment line `# unused options` instead of an empty line? It would help to better understand why those args are separated here.